react/features/settings/components/web/SettingsDialog.js

```diff
@@ -56,6 +62,116 @@ type Props = {
     dispatch: Function
 };
 
+/**
+ * Creates the styles for the component.
+ *
+ * @param {Object} theme - The current UI theme.
+ *
+ * @returns {Object}
+ */
+const styles = theme => {
+    return {
+        root: {
+            display: 'flex',
+            width: '100%',
+
+            '&.profile-pane': {
+                flexDirection: 'column'
+            },
+
+            '& .auth-name': {
+                marginBottom: theme.spacing(1)
```

Comment:
Done. 

I think you can pass a function instead of an array to mui `theme.spacing` so it could handle adding `px` when needed. Also the function could receive up to 4 arguments for top, right, bottom and left.